lib/suse/connect/client.rb

```diff
@@ -46,11 +46,8 @@ def register_product(product, install_release_package = true)
         service = activate_product(product, @config.email)
 
         System.add_service(service)
+        Zypper.install_release_package(product.identifier) if install_release_package
```

Comment:
AFAIK this was the only place where we called `Zypper.refresh_services`. Maybe you want to remove the method in `zypper.rb` too?